README.md

```diff
@@ -39,6 +40,13 @@ This demo illustrates tfautomv's core features:
 See [Getting started / Installation](https://padok-team.github.io/tfautomv/getting-started/installation/)
 for instructions.
 
+### Terraform version compatibility
+
+`tfautomv` uses your local Terraform installation. The following _Terraform version_ are supported:
+
+- from `0.13.x` to `1.0.x`, only with the `-output=commands` option, since _moved blocks_ are not supported in these versions
+- all Terraform version above `1.1.0`
+
```

Comment:
```suggestion
### Terraform version compatibility

The following versions of Terraform are supported:

- `1.1.x` and above by default
- `0.13.x` and above when using the `-output=commands` flag

```

Reply:
I think this wording is clearer with regards to which versions of Terraform work with which value of the `-output` flag.

I don't think this information needs its own section. I think a good place for it would be at the end of the `Usage` section in `README.md`:

```markdown
## Usage

See [Getting started / Tutorial](https://padok-team.github.io/tfautomv/getting-started/tutorial/)
for a hands-on guided introduction de tfautomv.

See [Usage](https://padok-team.github.io/tfautomv/usage/) for a list of
tfautomv's features.

The following versions of Terraform are supported:

- `1.1.x` and above by default
- `0.13.x` and above when using the `-output=commands` flag

## How it works
```